src/Components/Common/Button/Button.js

```diff
@@ -0,0 +1,7 @@
+import classes from './Button.module.scss';
+
+export const Button = ({buttonText, callbackClick}) => {
+  return (
+    <button onClick={callbackClick} className={classes.button}>{buttonText}</button>
```

Comment:
I could propose simple param names like:
onClick and text
So it will be:
```
<button onClick={onClick} className={classes.button}>{text}</button>

```

Reply:
Done!